packages/firestore/src/core/firestore_client.ts

```diff
@@ -297,6 +300,20 @@ export class FirestoreClient {
     return true;
   }
 
+  /**
+   * Checks that the client has not been shutdown. Ensures that other methods on
+   * this class cannot be called after the client is shutdown.
+   */
+  private checkClientRunning(): void {
```

Comment:
naming nit: I think `verifyClientRunning()` might make it a bit clearer that the method throws and be consistent with similar patterns in our code (e.g. https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/src/api/database.ts#L850)

And to use consistent terminology (reducing ambiguity), consider `verifyNotShutdown()`

Reply:
Just to double-check, did you see my `verifyNotShutdown()` suggestion? It looks like you didn't implement it and I'm not sure if that was an oversight or intentional. 😄